### ISSUE TITLE
what is the in label format？

### ISSUE THREAD

**Issue by menggege321** [Daniil-Osokin/lightweight-human-pose-estimation.pytorch#62]
Thank you very much for your work, but I have some problems in the process of using. I hope to get a reply: I use the annotations in coco format as the label directly, but why do I report errors when reading it, and the similar label ['img_upaths'] in it is not in coco format. What kind of label should I make next? Hope to get a reply, thank you!

**Comment by Daniil-Osokin** [comment]
Hi! Do you use [original annotations](http://cocodataset.org/#download)? Training script expects annotations converted from COCO to internal format. Here is [code](https://github.com/Daniil-Osokin/lightweight-human-pose-estimation.pytorch/blob/master/scripts/prepare_train_labels.py) for that.

**Comment by menggege321** [comment]
Thank you for your last reply. I've successfully debugged the code. But I'm on leave at home, and I don't have GPU training. I plan to run demo with the weight of the pre training you provided. However, the address written on your official website seems to be wrong. It's a web page instead of a download address. I hope to get your reply again. Thank you very much.

**Comment by menggege321** [comment]
I think you may need to compress into a compressed package to download, thank you!

**Comment by Daniil-Osokin** [comment]
Yeah, the model link sometimes doesn't start the download dialog for some reason. To download make a right mouse click on the link and select "Save as".

**Comment by Daniil-Osokin** [comment]
Hope, it helped.

**Comment by menggege321** [comment]
yes，thank you very much。
